lib/iiif_print/metadata.rb

```diff
@@ -28,7 +28,7 @@ def build_metadata
     def build_metadata_for_v2
       fields.map do |field|
         if field.name == :collection && member_of_collection?
-          viewable_collections = Hyrax::CollectionMemberService.run(work, @current_ability)
+          viewable_collections = Hyrax::CollectionMemberService.run(SolrDocument.find(work.id), @current_ability)
```

Comment:
Talk to me about this.

Reply:
`work` is a SolrHit, and the `Hyrax::CollectionMemberService#run` needs a SolrDocument